Web/Edubase.Web.UIUnitTests/Areas/Establishments/Controllers/EstablishmentControllerTests.cs

```diff
@@ -313,8 +313,7 @@ public async Task Estab_AddOrReplaceEstablishmentAddress()
             var result = await controller.AddOrReplaceEstablishmentAddressAsync(5, "test");
 
             Assert.NotNull(result);
-            //TODO - The above assert was added when converted to XUnit, before this the test asserted nothing.
-            //Figure out if anything else needs testing on this, or does it it just test the method completes without exception
```

Comment:
@ScottDawsonDfE -- Can we assume you've checked and this TODO no longer applies? :)

Reply:
;-) Lets assume there is nothing todo

There are improvements that can be made to the tests, but for replacing the existing tests (work I did this time last year), this is sufficient